x/staging/posts/types/msgs.go

```diff
@@ -184,11 +180,11 @@ func (msg MsgReportPost) ValidateBasic() error {
 	}
 
 	if strings.TrimSpace(msg.ReportType) == "" {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "report type cannot be empty")
+		return ErrReportEmptyType
 	}
 
 	if strings.TrimSpace(msg.Message) == "" {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "report message cannot be empty")
+		return ErrReportEmptyMessage
```

Comment:
I think it's better to use a more generic `ErrInvalidReport` and specify the error here